modules/dop/component-protocol/components/project-pipeline/myPage/tabsTable/pipelineTable/provider.go

```diff
@@ -651,37 +652,33 @@ func (p *PipelineTable) RegisterMoreOperationOp(opData OpMoreOperationsItemClick
 	id := string(opData.ClientData.ParentDataRef.ID)
 	switch opData.ClientData.DataRef.ID {
 	case "run":
-		_, err := p.ProjectPipelineSvc.Run(p.sdk.Ctx, deftype.ProjectPipelineRun{
+		_, err := p.ProjectPipelineSvc.Run(p.sdk.Ctx, &projectpipelinepb.RunProjectPipelineRequest{
 			PipelineDefinitionID: id,
-			ProjectID:            p.InParams.ProjectID,
-			IdentityInfo:         apistructs.IdentityInfo{UserID: cputil.GetUserID(p.sdk.Ctx)},
```

Comment:
will cause unauthorized access? or how you decided

Reply:
The code has the permission to take userinfo from CTX to verify